developer_documentation.md

```diff
@@ -119,6 +120,31 @@ The killer bunny does not exist in Bedrock Edition. Nonetheless, this is primari
+|:-----------------------:|:-----------------------:|:----------------------------:|:----------------------------------------------------------------------------------------:|
+|          `ash`          |            -            |             `ash`            |                              Not present in Bedrock Edition                              |
+|        `barrier`        |            -            |           `barrier`          |                     Present in Bedrock Edition, but not as a particle                    |
+|       `bubble_pop`      |            -            |         `bubble_pop`         |                              Not present in Bedrock Edition                              |
+|         `bubble`        |  `basic_bubble_manual`  |               -              | Modified version of the basic_bubble_manual particle is used to spawn in all block types |
+|     `crimson_spore`     |            -            |        `crimson_spore`       |                              Not present in Bedrock Edition                              |
+|    `damage_indicator`   |            -            |      `damage_indicator`      |                              Not present in Bedrock Edition                              |
+|        `dolphin`        | `dolphin_move_particle` |           `dolphin`          |    Present in Bedrock Edition, but cannot be spawned normally due to builtin variables   |
+|     `enchanted_hit`     |            -            |    `enchanted_hit_single`    |                              Not present in Bedrock Edition                              |
+|            -            |            -            |   `enchanted_hit_multiple`   |                  Used for playing multiple scattered particles on attack                 |
+|         `flash`         |            -            |            `flash`           |                              Not present in Bedrock Edition                              |
+|     `landing_honey`     |            -            |        `landing_honey`       |                              Not present in Bedrock Edition                              |
+|      `landing_lava`     |            -            |        `landing_lava`        |                              Not present in Bedrock Edition                              |
+| `landing_obsidian_tear` |            -            |    `landing_obsidian_tear`   |                              Not present in Bedrock Edition                              |
+|        `nautilus`       |    `conduit_particle`   |          `nautilus`          |    Present in Bedrock Edition, but cannot be spawned normally due to builtin variables   |
+|         `sneeze`        |            -            |           `sneeze`           |               Implemented in Bedrock Edition as a variant of redstone dust               |
```

Comment:
This one likely needs a note as we don't use it in `/particle` spawning, but we do use it for the panda itself sneezing.

Reply:
Resolved by 4525101